manifests/v3/sites/ldap.pp

```diff
@@ -28,21 +30,20 @@
 # @param idle_timeout
 #
 class freeradius::v3::sites::ldap (
-  String                $site_name           = 'default',
-  Boolean               $enable              = true,
-  Boolean               $include_listener    = true,
-  Simplib::Host         $listen_ip           = 'ALL',
-  Stdlib::Absolutepath  $confdir             = simplib::lookup( 'freeradius::confdir', {'default_value' => '/etc/raddb'} ),
-  String                $group               = simplib::lookup( 'freeradius::group', {'default_value' => 'radiusd'} ),
-  Boolean               $fips                = simplib::lookup('simp_options::fips', {'default_value' => false }),
-  Integer               $max_connections     = 16,
-  Integer               $lifetime            = 0,
-  Integer               $idle_timeout        = 30
+  String               $site_name        = 'default',
+  Boolean              $enable           = true,
+  Boolean              $include_listener = true,
+  Simplib::Host        $listen_ip        = 'ALL',
+  Stdlib::Absolutepath $confdir          = simplib::lookup( 'freeradius::confdir', {'default_value' => '/etc/raddb'} ),
+  String               $group            = simplib::lookup( 'freeradius::group', {'default_value' => 'radiusd'} ),
+  Integer              $max_connections  = 16,
+  Integer              $lifetime         = 0,
+  Integer              $idle_timeout     = 30
 ){
 
   include 'freeradius'
 
-  if $fips or $facts['fips_enabled'] {
+  if $facts['fips_enabled'] {
```

Comment:
IMHO the original implementation was correct.  That way the user is notified immediately that they have bad configuration.  As this change is written, they will find out after a system reboot once FIPS is applied.